src/main/java/com/exasol/adapter/document/json/JsonDateLoaderUdf.java

```diff
@@ -0,0 +1,16 @@
+package com.exasol.adapter.document.json;
+
+import com.exasol.adapter.document.AbstractDataLoaderUdf;
+import com.exasol.adapter.document.documentnode.json.JsonNodeVisitor;
+import com.exasol.adapter.document.mapping.PropertyToColumnValueExtractorFactory;
+import com.exasol.adapter.document.mapping.json.JsonPropertyToColumnValueExtractorFactory;
+
+/**
+ * This class is the UDF entry point for the data loading from JSON files.
+ */
+public class JsonDateLoaderUdf extends AbstractDataLoaderUdf<JsonNodeVisitor> {
```

Comment:
This seems to be: `JsonDataLoaderUdf`. Also rename the file and other references.